toml_testgen_support_test.go

```diff
@@ -1,3 +1,4 @@
+//go:generate go run github.com/toml-lang/toml-test/cmd/toml-test@master -copy ./tests
```

Comment:
This now always uses the latest available version, which might be in your interest as you can now generate tests for the new examples as soon as they come out.